portal/src/hook/usePivot.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { IPivotItemProps } from "@fluentui/react";
+
+export function usePivot(): {
+  hash: string;
```

Comment:
Please use a more specific name for this field (`selectedKey`?), since using URL fragment is an internal details of the hook.

A more descriptive name may be used for the hook too, e.g. `usePivotNavigation`.